docs/make.jl

```diff
@@ -1,53 +1,39 @@
 using Documenter
 using PowerAnalytics
-using Literate
+import DataStructures: OrderedDict
 
-folders = Dict()
 
-for (name, folder) in folders
-    for file in folder
-        outputdir = joinpath(pwd(), "docs/src/howto")
-        inputfile = joinpath(pwd(), "docs/src/$name/$file")
-        Literate.markdown(inputfile, outputdir)
-    end
-end
-if isfile("docs/src/howto/.DS_Store.md")
-    rm("docs/src/howto/.DS_Store.md")
-end
+pages = OrderedDict(
+    "Welcome Page" => "index.md",
+    "Tutorials" => Any["stub" => "tutorials/stub.md"],
+    "How to..." => Any["stub" => "how_to_guides/stub.md"],
+    "Explanation" => Any["stub" => "explanation/stub.md"],
+    "Reference" => Any[ 
+        "Public API" => "reference/public.md",
+        "Developers" => ["Developer Guidelines" => "reference/developer_guidelines.md",
+        "Internals" => "reference/internal.md"]]
+     
+)
+
 
 makedocs(
-    sitename = "PowerAnalytics.jl",
-    format = Documenter.HTML(
-        mathengine = Documenter.MathJax(),
-        prettyurls = get(ENV, "CI", nothing) == "true",
-    ),
     modules = [PowerAnalytics],
-    authors = "Clayton Barrows",
-    pages = Any[
-        "Introduction" => "index.md",
-        #"Quick Start Guide" => "qs_guide.md",
-        #"Logging" => "man/logging.md",
-        # "Operation Model" => "man/op_problem.md",
-        # "How To" => Any[
-        #     "Set Up Plots" => "howto/3.0_set_up_plots.md",
-        #     "Make Stack Plots" => "howto/3.1_make_stack_plots.md",
-        #     "Make Bar Plots" => "howto/3.2_make_bar_plots.md",
-        #     "Make Fuel Plots" => "howto/3.3_make_fuel_plots.md",
-        #     "Make Forecast Plots" => "howto/3.4_make_forecast_plots.md",
-        #     "Plot Fewer Variables" => "howto/3.5_plot_fewer_variables.md",
-        #     "Plot Multiple Results" => "howto/3.6_plot_multiple_results.md",
-        # ],
-        #"Simulation Recorder" => "man/simulation_recorder.md",
-        #"Model References" => Any["Hydro Models" => "ref/hydro.md"],
-        "API" => Any["PowerAnalytics" => "api/PowerAnalytics.md"],
-    ],
+    format = Documenter.HTML(
+        prettyurls = haskey(ENV, "GITHUB_ACTIONS"),
+        size_threshold = nothing,),
+    sitename = "PowerAnalytics.jl",
+    authors = "Gabriel Konar-Steenberg <gabriel.konarsteenberg@nrel.gov>, cbarrows <clayton.barrows@nrel.gov>",
```

Comment:
```suggestion
    authors = "Gabriel Konar-Steenberg and Clayton Barrows
```
following the format of https://github.com/NREL-Sienna/PowerSystems.jl/blob/be072caaaf17f83a6109a8d3f20f1c2acc517e61/docs/make.jl#L140 unless there's a reason not to?